docs/overview.rst

```diff
@@ -86,70 +94,255 @@ Important services in the diagram:
 * **Collector:** Collects incoming crash reports via HTTP POST. The collector
   we use is called `Antenna <https://antenna.readthedocs.io/>`_. It saves
   crash data to AWS S3.
-* **Processor:** Extracts data from minidumps, generates crash signatures,
-  performs other analysis, and saves everything as a processed crash.
+
+* **Processor:** Processes crashes and extracts data from minidumps, generates
+  crash signatures, performs other analysis, and saves everything as a processed
+  crash.
+
 * **Webapp (aka Crash Stats):** Web user interface for analyzing crash data.
-* **Crontabber:** Runs hourly/daily/weekly tasks for analyzing and processing
-  data.
 
-The processor stores crash data in several crash storage destinations:
+* **Crontabber:** Runs periodic housekeeping tasks.
+
+
+Let's take a tour through the crash ingestion pipeline!
+
+
+A tour through the crash ingestion pipeline
+===========================================
+
```

Comment:
If you like being funny and popular you could put another line here: "This is the tour through the crash ingestion pipeline"